expenses/views/reports.py

```diff
@@ -118,4 +125,5 @@ def report_run(request, slug):
             "time": end_time - start_time,
             "postfields": postfields,
         },
+        content_type
```

Comment:
`text/csv` files are downloaded with random filenames by Firefox; a custom filename should be provided to improve user experience.